crates/vault-registry/src/types.rs

```diff
@@ -699,6 +699,14 @@ impl<T: Config> RichVault<T> {
         }
     }
 
+    pub fn has_no_outstanding_wrapped_tokens(&self) -> Result<bool, DispatchError> {
+        let total_wrapped_tokens = self
+            .backed_tokens()?
+            .checked_add(&self.to_be_replaced_tokens())?
```

Comment:
I'm not sure this should be included here - to_be_replaced_tokens is not like issued/to_be_issued/to_be_redeemed, in that they don't represent an obligation - it's simply bookkeeping to keep track how many tokens the vault asked to be replaced for. 

Actually, now that I think about it, what we should probably do is to withdraw the replace request if there is one in the liquidation function. This will free up a little bit of collateral for the vault as well

Reply:
It's not as straightforward as calling `withdraw_replace()` in the `replace` pallet, since `replace` depends on the `vault-registry` rather than the other way around. Should I move the function [body](https://github.com/interlay/interbtc/blob/825030f2caaf741c811568b24d42d1a2547810f7/crates/replace/src/lib.rs#L361) inside the vault registry?

Reply:
Yea I think we can move [these lines](https://github.com/interlay/interbtc/blob/825030f2caaf741c811568b24d42d1a2547810f7/crates/replace/src/lib.rs#L364-L372) to a `withdraw_replace_request` function in the vault registry